### PR TITLE
TEAMNADO-3886: Change to capacity object

### PR DESCRIPTION
**Pull request by cowanrc**

### CODE REVIEW

File: src/components/ProductsTable/__tests__/ProductsTable.test.tsx [hunk 1]
```diff
@@ -413,16 +413,14 @@ describe('ProductsTable', () => {
         sku: 'RH123',
         quantity: 2,
         productLine: 'consonants',
-        serviceLevel: 'Standard',
-        unitOfMeasure: { name: 'Mock1', quantity: 'unlimited' }
+        serviceLevel: 'Standard'
```
Comment (jlgre) [inline src/components/ProductsTable/__tests__/ProductsTable.test.tsx:416]: Should this have the capacity in it still? I think this should just be a rename

Comment (cowanrc) [reply src/components/ProductsTable/__tests__/ProductsTable.test.tsx:416]: > Should this have the capacity in it still? I think this should just be a rename

I believe these just slipped through the cracks when the story of removing Unit of Measure from Products Table was pushed out. Neither UoM or Capacity should be in Products Table.

Comment (jlgre) [reply src/components/ProductsTable/__tests__/ProductsTable.test.tsx:416]: Ack - good call

**Review by jlgre — APPROVED** [review]
Looks good

**Pull request merged**